### ISSUE TITLE
When a PR goes completely green, and is marked for merging, add it to the train

### ISSUE THREAD

**Issue by lastquestion** [lastquestion/steward-bot#10]
the situation is:

1. your train has a PR waiting on build
2. another PR that was not part of the original train, because e.g. it was failing build, passes green (maybe the developer pushed a fix)
3. it builds faster then the original train's builds for some reason
4. it is now green, and really it could be added to the train and get merged at once

it's similar to the change made by #3.
